0000-experiment-dataframe.md

```diff
@@ -0,0 +1,320 @@
+
+#### Example 1: Parallel experiment.
+
+```python
+pexp = ParallelExperiment(
+  [
+    T1(qubit=0, delays=delays),
+    T1(qubit=1, delays=delays),    
+  ],
+  backend=ibm_washington,
+)
+experment_data = pexp.run()
+
+experiment_data.analysis_results(dataframe=True)
+```
+
+Analysis results:
+
+|          | name   | value  | quality | device_components |  backend        | creation_datetime          | experiment | ... |
+|----------|--------|--------|---------|-------------------|-----------------|----------------------------|------------|-----|
+| 03fe91s1 | T1     | 100e-6 | good    | [Q0]              |  ibm_washington | 2022-11-22 03:55:30.313872 | T1         | ... |
+| 1c4ad2a1 | T1     | 120e-6 | good    | [Q1]              |  ibm_washington | 2022-11-22 03:55:30.313872 | T1         | ... |
+
+
+#### Example 2: Batch experiment.
+
+```python
+bexp = BatchExperiment(
+  [
+    T1(qubit=0, delays=delays),
+    T2Ramsey(qubit=0, delays=delays),    
+  ],
+  backend=ibm_washington,
+)
+experment_data = bexp.run()
+
+experiment_data.analysis_results(dataframe=True)
+```
+
+Analysis results:
+
+|          | name   | value  | quality | device_components |  backend        | creation_datetime          | experiment | ... |
+|----------|--------|--------|---------|-------------------|-----------------|----------------------------|------------|-----|
+| 17636b47 | T1     | 100e-6 | good    | [Q0]              |  ibm_washington | 2022-11-22 03:58:36.523230 | T1         | ... |
+| 2f50a741 | T2star |  80e-6 | good    | [Q0]              |  ibm_washington | 2022-11-22 03:58:36.523230 | T2Ramsey   | ... |
+
+
+#### Example 3: Repeating the same experiment.
+
+```python
+exp = T1(qubit=0, delays=delays)
+
+trend_data = []
+for _ in range(2):
+  experment_data = exp.run()
+  trend_data.append(experment_data)
+  time.sleep(3600)
+
+# combine trend data
+...
+```
+
+Analysis results (combined):
+
+|          | name   | value  | quality | device_components |  backend        | creation_datetime          | experiment | ... |
+|----------|--------|--------|---------|-------------------|-----------------|----------------------------|------------|-----|
+| 03fe91s1 | T1     | 100e-6 | good    | [Q0]              |  ibm_washington | 2022-11-22 04:01:30.313872 | T1         | ... |
+| 1c4ad2a1 | T1     | 103e-6 | good    | [Q0]              |  ibm_washington | 2022-11-22 05:01:14.123201 | T1         | ... |
+
+
+#### Complement
+
+As these examples show, experimentalist can distinguish the outcomes of batch (with `experiment`) and parallel experiments (with `device_components`). Even though there are multiple entries for the identical parameter (example 3), they have different result IDs and creation date times. In other words, one can sort the entries by `creation_datetime` and get the time-ordered sequence of `value` for the trend analysis:
```

Comment:
These should have different experiment_ids too

Reply:
Thanks. Updated in [8454c5e](https://github.com/Qiskit/rfcs/pull/28/commits/8454c5eeb0a4ad360ef7966b56c5e67e1471b369)